accepted/install-locations.md

```diff
@@ -0,0 +1,239 @@
+# Install location search for .NET Core
+
+.NET Core has several cases where it needs to search for the install location of the runtime and SDKs. This document described the proposed algorithm for how this search will be done on all the supported platforms.
+
+## .NET Core install layout
+The content of a .NET Core install location always follows the same layout. The directory structure is the same on all platforms, file names may differ to conform with the platform specific requirements. (Below sample uses file names for Windows):
+* `./dotnet.exe` - the "muxer" which is used to execute apps and issue dotnet CLI commands.
+* `./host/fxr/<version>/hostfxr.dll` - this library contains all the logic to resolve framework references and select SDK. Regardless of application, the latest available version is used.
+* `./shared` - contains shared frameworks.
+  * `./Microsoft.NETCore.App/<version>` - the root framework which contains the .NET Core runtime. This framework contains `hostpolicy.dll` which is the entry point called by the `hostfxr` to run the app. Applications can choose which version to use.
+  * `./<FrameworkName>/<version>` - any other shared framework. Applications can choose which version to use.
+* `./sdk/<version>` - only installed when .NET Core SDK is installed - contains the SDK tools. Users can choose which version to use.
+
```

Comment:
.NET Core 3.0 will also introduce `./packs/<Frameworkname>.Ref/` (followed by a version?).

Reply:
Thanks, this is not meant to be a complete layout of the install location, it's here to illustrate what important pieces are located there. I will a note that the list is not complete.